module/config.mjs

```diff
@@ -3470,9 +3470,9 @@ preLocalize("cover");
  * @deprecated since v10
  */
 DND5E.trackableAttributes = [
-  "attributes.ac.value", "attributes.init.bonus", "attributes.movement", "attributes.senses", "attributes.spelldc",
-  "attributes.spellLevel", "details.cr", "details.spellLevel", "details.xp.value", "skills.*.passive",
-  "abilities.*.value"
+  "attributes.ac.value", "attributes.init.bonus", "attributes.movement", "attributes.senses",
+  "attributes.spell.attack", "attributes.spell.dc", "attributes.spellLevel", "details.cr",
+  "details.spellLevel", "details.xp.value", "skills.*.passive", "abilities.*.value"
```

Comment:
Seems a bit weird that `spellLevel` is under `details` for NPCs (and `attributes.spellLevel` doesn't seem to exist?). Maybe we should move it under `attributes.spell.level` for consistency.